apps/pie-docs/src/foundations/motion/overview/motion-principles.json

```diff
@@ -0,0 +1,32 @@
+{
+    "items": [
+        {
+            "headingText": "Purposeful",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/purposeful.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/purposeful_narrow.svg",
+            "text": "Motion always feels intentional and appropriate for the context it’s being used in. It aligns with the user's goals and commands focus without causing unnecessary distractions.",
+            "alt": "Bright, abstract illustration depicting purposeful."
+        },
+        {
+            "headingText": "Smooth",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/smooth.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/smooth_narrow.svg",
+            "text": "Motion mimics real-world physics to always look and feeling fluid, natural and seamless. Motion is predictable and tells a story without fee jarring or out of place in our products.",
+            "alt": "Bright, abstract illustration depicting smoothness."
+        },
+        {
+            "headingText": "Responsive",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/responsive.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/responsive_narrow.svg",
+            "text": "Motion informs and guides our users by providing visual feedback and driving a users attention. It reinforces hierarchy between elements and is timely to avoid feeling intrusive or sluggish.",
```

Comment:
```suggestion
            "text": "Motion informs and guides our users by providing visual feedback and driving a user's attention. It reinforces hierarchy between elements and is timely to avoid feeling intrusive or sluggish.",
```